apps/src/lib/levelbuilder/lesson-editor/LevelTokenDetails.jsx

```diff
@@ -135,34 +136,34 @@ export class UnconnectedLevelTokenDetails extends Component {
 
   handleAddVariant = () => {
     this.props.addVariant(
-      this.props.lessonGroupPosition,
-      this.props.lessonPosition,
+      this.props.activityPosition,
```

Comment:
Variants are only really used to swap out a level mid year if something is wrong with the level right? Are we ok with engineering needed to be involved if this case comes up?

Reply:
Good point...I don't have a better solution for that scenario, so since this one is working, it seems better to keep it around. I do still think we can get rid of level variant experiments, but that's a separate issue.